### ISSUE TITLE
HPKP - Public Key Pinning id points to the wrong entry in chrome status

### ISSUE THREAD

**Issue by martinsuchan** [MicrosoftEdge/Status#449]
[This page shows](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/draganddropdirectories/?q=public%20key%20pinning) that HTTP Public Key Pinning (HPKP) is not supported in Firefox.  

It works in Firefox since v35.
http://caniuse.com/#feat=publickeypinning
https://developer.mozilla.org/en/docs/Web/Security/Public_Key_Pinning

**Comment by ghost** [comment]
The status is fetched from https://www.chromestatus.com/features/4669935557017600. It needs to be updated by Chromium team on their side. You need to file a bug on [chromium-dashboard](https://github.com/GoogleChrome/chromium-dashboard).

**Comment by stof** [comment]
@nextPrime the chromium-dashboard status you linked is about the HPKP _reporting_ feature, not about HPKP itself.

**Comment by ghost** [comment]
@stof You're right. I didn't notice that. So https://github.com/MicrosoftEdge/Status/blob/production/status.json#L4182 should be changed. Another thing is since Firefox supports HPKP blocking but not reporting, should the status page reports HPKP is supported or not supported in Firefox? Unfortunately Edge status page doesn't indicate partial support.

**Comment by SaschaNaz** [comment]
Shouldn't this be directly fetched from [Firefox status page](https://github.com/mozilla/platform-status/)?

**Comment by molant** [comment]
There is no entry to track support for HPKP in Chrome status. We should remove the id and put the data manually.
